openedx/core/djangoapps/courseware_api/serializers.py

```diff
@@ -119,6 +119,8 @@ class CourseInfoSerializer(serializers.Serializer):  # pylint: disable=abstract-
     is_mfe_special_exams_enabled = serializers.BooleanField()
     is_mfe_proctored_exams_enabled = serializers.BooleanField()
     user_needs_integrity_signature = serializers.BooleanField()
+    course_wide_js = serializers.ListField(source='course.course_wide_js')
+    course_wide_css = serializers.ListField(source='course.course_wide_css')
```

Comment:
What does `serializers.ListField` do if there's a NULL value - will it return an empty list? Since the migration will set all existing courseoverviews to have a NULL value for this field (I think), I just want to make sure that the API will still return a consistent list data type (an empty list) for those courses that already existed and haven't specified a list of resources yet.

Reply:
Thats a good point, `serializers.ListField` will return a `null` value if the value null. I agree its best to return an empty list to keep it consistent.
I made [a change](https://github.com/edx/edx-platform/pull/28411/commits/1a25d7b98f6ac0ba6354270e46581b7f81e0f14e#diff-87609bf105866fbc6b3da8aa0761c3748f69ca3f3909fe44271692e3a43794c5R19) to the model (and migrations) in the fourth commit to give the fields a default value instead of allowing `null`. And the `xblock.fields.List` used in the [`CourseFields`](https://github.com/edx/edx-platform/pull/28411/files#diff-5e5a9a90f02439d6bcd1f65bc96da133eedae87338ba248f7bd49a4752fbbaf7R999) implicitly get a default value of `[]` so it should be fine there as well.